app/components/cloud-call.js

```diff
@@ -110,7 +94,6 @@ export default Component.extend({
       if (!shouldRender) return;
```

Comment:
The easier vanilla-js way to do it is to declare the var that starts the render outside of the component scope, assign it on didInsertElement, then call it from the scroll function.  something like

```
let render;
Component.extend({
  didInsertElement() {
    render = () => {
      // render function
    }
  }
  handleScroll() {
    if (shouldRender) render()
  }
})
```